resources/multisite_drupal_standard.make

```diff
@@ -661,24 +661,26 @@ projects[rules][patch][] = https://www.drupal.org/files/issues/2019-01-25/302844
 ; https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-2160
 projects[rules][patch][] = https://www.drupal.org/files/issues/file_events-826986-31_0.patch
 
+; https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-2615
+; We use the dev version of the module to be able to run the module simpletes.
 projects[scheduler][subdir] = "contrib"
-projects[scheduler][version] = 1.5
+projects[scheduler][download][type] = "git"
+projects[scheduler][download][url] = "https://git.drupalcode.org/project/scheduler.git"
+projects[scheduler][download][revision] = "89707ba3affa72beea0b428230e61f4c5a0c1283"
 
+; https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-2615
+; We use the dev version of the module to be able to run the module simpletes.
```

Comment:
Missing 't' at the end of this line as well.

Reply:
Changed to simpletest